README.md

```diff
@@ -4,4 +4,28 @@ Launch Program.
 
 Expected output: "Hello World!"
 
-Enjoy
+## FUNctions
+
+A C++ function **definition** consists of a function header and a function body. A function **definition** tells the compiler about a function's name, return type, and parameters. A function definition provides the actual body of the function.
+
+Here are all the parts of a function −
+
+_Return Type_ − A function may return a value. The return_type is the data type of the value the function returns. Some functions perform the desired operations without returning a value. In this case, the return_type is the keyword void.
+
```

Comment:
Good. Remember, the phrase/term "return type" is used in many languages. So that's why we call it "return type" instead of "return_type." That's because when we say "return_type", that's kinda written like a variable would be written, using snake_case (note that JS developers usually use camelCase for variables instead of snake_case for variables). 

As such, this sentence: `In this case, the return_type is the keyword void.` should read `In this case, the return type is the keyword void.`

Once this is addressed you can reply to this comment and say "Addressed." and then paste in the hash number of the specific commit you used to address this comment/feedback. That way, we keep track of exactly how you're addressing your PR's feedback from other developers.

Reply:
The hash number of the commit is available in Github next to the commit. There's a little button to copy the commit hash. It will look something like abc123jkljkvhk13890 when you paste it in to the comment. So you'll fix this with a commit and reply as such: Addressed. hashy123abcjfldks7897thing.